torch/csrc/jit/OVERVIEW.md

```diff
@@ -154,26 +158,26 @@ Finally, nodes can have extra pieces of information assigned to them, which are
 
 Graphs in the JIT are in single-static assignment (SSA) form, meaning that each Value has precisely one defining Node that can be looked up directly from the Value (`Node* n = v.node()`).
 
-**Ownership Model** Blocks, Nodes, and Values are _owned_ by the Graph they appear in and may only appear in a single Graph. This is enforced by assertions in the API. Creation and deletion of Block, Node, and Value objects is done via methods on Graph objects (e.g. `Graph::create`,  `Node::addOutput`, or `Node::addBlock`). This API also enforces certain consistency properties. For instance, `Node::destroy` removes a Node, but it is only valid to call this function if the Values produced by this node are no longer used, which can be accomplished using other functions such as `Value::replaceAllUsesWith`.
+**Ownership Model** Blocks, Nodes, and Values are _owned_ by the Graph they appear in and may only appear in a single Graph. This is enforced by assertions. Creation and deletion of Nodes is done via methods on Graph objects (e.g. `Graph::create`). The code also enforces certain consistency properties. For instance, `Node::destroy` removes a Node, but it is only valid to call this function if the Values produced by this Node are no longer used, which can be accomplished using other functions such as `Value::replaceAllUsesWith`.
```

Comment:
I think "is done via methods on Graph objects" is misleading since there is no method on a Graph that can create or delete a Node or Value.  I added another sentence to discuss Block and Value objects. LGTY?